examples/ota-requestor-app/esp32/main/main.cpp

```diff
@@ -73,6 +74,33 @@ static void InitServer(intptr_t context)
 
 } // namespace
 
+// Test mode operation
+void OnStartDelayTimerHandler(Layer * systemLayer, void * appState)
+{
+    OTARequestor * gRequestorCore = static_cast<OTARequestor *>(appState);
+    uint32_t softwareVersion;
+    NodeId nodeId;
+    FabricIndex fabIndex;
+    EndpointId endpointId;
+    esp_err_t err = OTARequestorNVSHelper::ReadProviderLocation(nodeId, fabIndex, endpointId);
```

Comment:
I'm hoping with persistent storage working later on in the OTA Requestor core, this and the calling of `TestModeSetProviderParameters` will not be needed later.

Reply:
Yup